tsconfig-google.json

```diff
@@ -4,7 +4,6 @@
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
     "declaration": true,
-    "esModuleInterop": true,
```

Comment:
Do we need to disable synthetic default imports as well?  Or am I getting my compiler flag wires crossed?

Reply:
No.. esModuleInterop implicitly enables synthetic imports.  See here: https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
We explicitly turn it on here tho: https://github.com/google/ts-style/pull/183/files#diff-19e170b73889957f2d2c69de5a7a5d7aL3

Reply:
D'oh.

Reply:
#184